readme.md

```diff
@@ -80,6 +81,21 @@ referred to as `Playing the Hitler card`.
 
 ***
 
+### [Parkinson's law](https://en.wikipedia.org/wiki/Parkinson%27s_law)
+
+> Parkinson's law is the adage that "work expands so as to fill the time available for its completion". 
```

Comment:
No need to put the introductory text, just mention the adage (without double quotes). See other adages for ref